Source/Core/UICommon/GameFile.cpp

```diff
@@ -149,13 +155,54 @@ bool GameFile::IsValid() const
   return true;
 }
 
+void GameFile::UpdateCover()
+{
+  if (!m_cover.buffer.empty() || !Config::Get(Config::MAIN_USE_GAME_COVERS))
+    return;
+
+  Common::HttpRequest request;
+
+  std::string region_code;
+
+  switch (m_region)
+  {
+  case DiscIO::Region::NTSC_J:
+    region_code = "JA";
+    break;
+  case DiscIO::Region::NTSC_U:
+    region_code = "US";
+    break;
+  case DiscIO::Region::NTSC_K:
+    region_code = "KO";
+    break;
+  case DiscIO::Region::PAL:
+  case DiscIO::Region::Unknown:
+    region_code = "EN";
```

Comment:
You should use the user's GC/Wii language preference instead of assuming English.